src/main/java/com/minecolonies/core/entity/ai/minimal/EntityAISickTask.java

```diff
@@ -245,10 +227,15 @@ private IState applyCure()
             return CHECK_FOR_CURE;
         }
 
-        final List<ItemStack> list = IColonyManager.getInstance().getCompatibilityManager().getDisease(citizen.getCitizenDiseaseHandler().getDisease()).getCure();
+        if (citizen.getCitizenDiseaseHandler().getDisease() == null)
```

Comment:
save disease to variable, reuse it then later in this method instead of doing the long query

Reply:
This is no long query, it's a straight up getter.